### PR TITLE
First publish of roguepullreqest programmer dvorak planck layout

### PR DESCRIPTION
**Pull request by roguepullrequest**
## Description
A modified version of programmer dvorak for the planck keyboards. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/planck/keymaps/roguepullrequest/config.h [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#define DEVICE_VER	0x0004
```
Comment (drashna) [inline keyboards/planck/keymaps/roguepullrequest/config.h]: This isn't needed, and should be removed. 

```suggestion
```

File: keyboards/planck/keymaps/roguepullrequest/config.h [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+
+
+
+#ifdef AUDIO_ENABLE
+	#define STARTUP_SONG SONG(RICK_ROLL)
+	
+	#define DEFAULT_LAYER_SONGS { \
```
Comment (drashna) [inline keyboards/planck/keymaps/roguepullrequest/config.h]: If you're not using this, you could safely remove them

File: keyboards/planck/keymaps/roguepullrequest/keymap.c [hunk 3]
```diff
@@ -0,0 +1,187 @@
+#include QMK_KEYBOARD_H
+#include "planck.h"
```
Comment (drashna) [inline keyboards/planck/keymaps/roguepullrequest/keymap.c]: The above include handles this, actually.
```suggestion
```

File: keyboards/planck/keymaps/roguepullrequest/keymap.c [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+#include "eeconfig.h"
```
Comment (drashna) [inline keyboards/planck/keymaps/roguepullrequest/keymap.c]: This isn't needed:
```suggestion
```

File: keyboards/planck/keymaps/roguepullrequest/keymap.c [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+
+#ifdef AUDIO_ENABLE
+	#include "audio.h"
```
Comment (drashna) [inline keyboards/planck/keymaps/roguepullrequest/keymap.c]:
```suggestion
```

File: keyboards/planck/keymaps/roguepullrequest/keymap.c [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+	#include "song_list.h"
```
Comment (drashna) [inline keyboards/planck/keymaps/roguepullrequest/keymap.c]:
```suggestion
```

File: keyboards/planck/keymaps/roguepullrequest/readme.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
+# Programmer Dvorak for the Planck (Light)
+
+![LAYOUT IMAGE v0.1](layout.jpg?raw=true "Planck Layout")
```
Comment (noroadsleft) [inline keyboards/planck/keymaps/roguepullrequest/readme.md]: This line should be deleted.

```suggestion
```

**Comment by drashna** [comment]
Awesome, thanks!

**Pull request merged**